CuratedContent/Jupyter4Science.md

```diff
@@ -0,0 +1,48 @@
+# Jupyter4Science: Better Practices for using Jupyter Notebooks for Science
```

Comment:
I made up a title.  Please confirm or suggest something else.

Reply:
Since she didn't comment or change the text, I assume this is okay with @nicole-brewer